src/platform-includes/enriching-events/attach-viewhierarchy/android.mdx

```diff
@@ -4,6 +4,46 @@
 </application>
 ```
 
+### Customize View Hierarchy Capturing
+
+<Note>
+
+Requires SDK version `6.24.0` or higher.
+
+</Note>
+
+On Android capturing view hierarchy can be an resource-intense operation, thus capturing is limited to one view hierarchy every 2 seconds using a debouncing mechanism. This behavior can be overruled by supplying a `BeforeCaptureCallback` for view hierarchies in the `SentryAndroidOptions`.
+
```

Comment:
```suggestion
Because capturing view hierarchy can be a resource-intensive operation on Android, it's limited to one view hierarchy every 2 seconds using a debouncing mechanism. This behavior can be overruled if you supply a `BeforeCaptureCallback` for view hierarchies in the `SentryAndroidOptions`.

```